orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/AbstractClusterWideClouddriverTaskSpec.groovy

```diff
@@ -142,6 +143,25 @@ class AbstractClusterWideClouddriverTaskSpec extends Specification {
 
   }
 
+  def 'cluster wide task should use moniker if available or else fallback on frigga'() {
```

Comment:
If you attach `@Unroll` here each `where` is given its own test. Can help with making a failing test easier to debug.

Reply:
You can also embed `cluster` & `expected` into the test name like this: http://spockframework.org/spock/javadoc/1.0/spock/lang/Unroll.html